sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -2339,4 +2339,22 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
       Literal.create(Seq(Float.NaN, null), ArrayType(FloatType)),
       Literal.create(Seq(Float.NaN, null, 1f), ArrayType(FloatType))), true)
   }
+
+  test("SPARK-36740: ArrayMin/ArrayMax/SortArray add comment and unit test for " +
```

Comment:
In this case, could you describe the content of this test case instead of PR title?

Reply:
Done